### ISSUE TITLE
Late Oxfordian data points displayed in black

### ISSUE THREAD

**Issue by vjpsyverson** [paleobiodb/navigator#131]
Some Jurassic collections (e.g. https://paleobiodb.org/classic/basicCollectionSearch?collection_no=84308) display as black points (rather than blue) in Navigator. As far as I can tell, all of them have time intervals overlapping the Late Oxfordian. Is this expected behavior?

see: [(https://paleobiodb.org/navigator/#/4fe78575)]

**Comment by mmcclenn** [comment]
No, this is not correct behavior. At some point, I will have to dig into the Navigator code and try to figure out how to fix problems like this.

**Comment by mmcclenn** [comment]
Thanks for letting us know about this!

**Comment by Olly4k** [comment]
Hey, I'm a university student looking to contribute to an open source project. Would this issue be available for me to have a look at?
